src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-table/TranscriptsTable.scss

```diff
@@ -0,0 +1,93 @@
+@import 'src/styles/common';
+
+.table {
+  display: grid;
+  grid-template-columns: 1fr 3fr 1fr 1fr 2fr;
+  grid-template-rows: auto;
+  min-width: 1900px;
+}
+
+.tableRow {
+  border-bottom: 1px solid $dark-grey;
+}
+
+.tableCell {
+  display: inline-block;
+  padding: 8px 4px;
+}
+
+.tableBody {
+  padding-top: 20px;
+}
+
+.tableHeader {
+  color: $grey;
+
+  .tableRow {
+    border-bottom: 1px solid #8d621c;
+  }
+}
+
+.tableLeftWing {
+  grid-area: 1 / 1 / auto / 2;
+
+  .tableHeader {
+    text-align: center;
+  }
+
+  .tableBody {
+    text-align: right;
+
+    .tableRow {
+      border-bottom: none;
+    }
+  }
+}
+
+.tableMiddle {
+  grid-area: 1 / 2 / auto / 3;
+
+  .tableRow {
+    display: inline-grid;
+    grid-template-rows: auto;
```

Comment:
No-no-no, the class tells us that it’s a table row. A row should not consist of rows; that's what columns do.

Reply:
Ah I forgot to remove this one. Thanks for spotting it.

Reply:
:heavy_check_mark:

Reply:
Also, `inline-grid` -> `grid`.